va/va.c

```diff
@@ -34,14 +34,14 @@
 #include <assert.h>
 #include <stdarg.h>
 #include <stdio.h>
-#include <stdlib.h>
 #include <string.h>
 #if defined(_WIN32)
 #include "compat_win32.h"
 #define DRIVER_EXTENSION    "_drv_video.dll"
 #define DRIVER_PATH_STRING  "%s\\%s%s"
 #define ENV_VAR_SEPARATOR ";"
 #else
+#include <stdlib.h>
```

Comment:
I don't think we need this change here. You might use it for testing that on Windows all is getting build and works as expected, but code here was good as is: stdlib.h was defined before compat_win32.h. I think there is no need to change this file. Revert.